vizro-core/docs/pages/user_guides/container.md

```diff
@@ -1,13 +1,31 @@
 # How to use containers
 
-This guide shows you how to use containers to group your page components into sections and subsections.
+This guide shows you how to use containers to group your components into sections and subsections within the page.
 
+A [`Container`][vizro.models.Container] complements the concept of a [`Page`][vizro.models.Page], and the two models have almost identical arguments.
+ [`Page.layout`](layouts.md) provides a way to structure the overall layout of the page, and a `Container` allows for more granular control within a specific section of that page.
+
+While there is currently no apparent difference in rendering, additional functionality will be added to the `Container` soon (e.g. controls specific to that container),
+enhancing the ability to manage related components.
+
+## When to use containers
+In general, any arbitrarily granular layout can already be achieved using [`Page.layout`](layouts.md) alone and is our
+recommended approach if you just want to arrange components on a page with consistent row and/or column spacing.
+However, there are a few cases where you might want to use a `Container` instead:
+
+- If you want to split up your grid into subgrids to organize components together
+- If you want to add a title to your subgrids
+- If you want different row and column spacing between subgrids
```

Comment:
This point about different spacing is a good one and not something I had thought of before 👍